velox/exec/tests/utils/HiveConnectorTestBase.h

```diff
@@ -281,6 +282,8 @@ class HiveConnectorSplitBuilder {
   dwio::common::FileFormat fileFormat_{dwio::common::FileFormat::DWRF};
   uint64_t start_{0};
   uint64_t length_{std::numeric_limits<uint64_t>::max()};
+  facebook::velox::connector::hive::FileProperties properties_ =
```

Comment:
`facebook::velox::connector::hive::FileProperties properties_;` will call the default constructor.

Reply:
Done, thanks!